lib/cerner/oauth1a/access_token_agent.rb

```diff
@@ -171,12 +158,88 @@ def convert_to_http_uri(access_token_url)
             raise ArgumentError, 'access_token_url is invalid'
           end
         end
-        unless uri.is_a? URI::HTTP
-          raise ArgumentError, 'access_token_url must be an HTTP or HTTPS URI'
-        end
+        raise ArgumentError, 'access_token_url must be an HTTP or HTTPS URI' unless uri.is_a?(URI::HTTP)
         uri
       end
-    end
 
+      # Internal: Prepare a request for #retrieve
+      def retrieve_prepare_request(timestamp, nonce, accessor_secret, principal)
```

Comment:
Minor, but for readability I feel like these functions would be better like: `prepare_retrieve_request` and `handle_retrieve_response`.